skimage/io/tests/test_pil.py

```diff
@@ -97,6 +97,17 @@ def test_imread_truncated_jpg():
                   os.path.join(data_dir, 'truncated.jpg'))
 
 
+def test_jpg_quality_arg():
+    chessboard = np.load(os.path.join(data_dir, 'chessboard_GRAY_U8.npy'))
+    with NamedTemporaryFile(suffix='.jpg') as jpg:
```

Comment:
@blink1073 @stefanv it looks like `NamedTemporaryFile` is a bit more hassle than it's worth, with default use being rather finicky for our purpose (a filename that can be written to but will be deleted when the test is done). How do you feel about a context manager, perhaps in `skimage.util`? Example usage:

```python
with temporary_file('.tif') as filename:
    do_test_roundtrip(filename, image)
```

The context manager would ensure that the file is properly created with `NamedTemporaryFile` using `suffix` and `delete=False`, closed, and then deleted with `os.remove` when we exit the scope. Thoughts?